pages/domains/domains_eligibility.md

```diff
@@ -51,13 +51,13 @@ Read more about authorizing officials for:
 - [Executive branch federal agencies](#authorizing-officials-for-executive-branch-federal-agencies)
 - [Judicial branch federal agencies](#authorizing-officials-for-judicial-branch-federal-agencies)
 - [Legislative branch federal agencies](#authorizing-officials-for-Legislative-branch-federal-agencies)
```

Comment:
new anchor link should be `#authorizing-officials-for-legislative-branch-federal-agencies`